Maple2.Server.Game/Util/XmlParseUtil.cs

```diff
@@ -10,6 +10,10 @@ public static class XmlParseUtil {
             return parameters;
         }
 
+        if (xmlString[1] == ' ') {
```

Comment:
you could use `Trim()` which is more robust

```cs
xmlString = xmlString.Trim()
```

Reply:
I don't think Trim here would work.

The problem isn't the first character, it's the 2nd. This is how the string is formatted in the item XML:
```< v fieldID="81000003" portalDurationTick="180000" maxCount="20" />```

That space between '<' and 'v' breaks loading it as an XML.